### PR TITLE
Add dogechain mainnet & testnet

### PR DESCRIPTION
**Pull request by Huuuulk**

### CODE REVIEW

**Review by ligi — CHANGES_REQUESTED** [review]
short name must be unique - but found `dogechain` more than once

File: _data/chains/eip155-2000.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "Doge Chain Mainnet",
```
Comment (abrahamcruise321) [inline _data/chains/eip155-2000.json:2]: Doge Chain -> Dogechain

**Comment by HulkChou8** [comment]
@ligi Need to review again, thanks.

**Comment by HulkChou8** [comment]
@ligi Need to review again, thanks.

**Comment by Huuuulk** [comment]
@ligi Need to review again, thanks.

**Review by ligi — APPROVED** [review]

**Review by Huuuulk — COMMENTED** [review]
dogechain logo

**Pull request closed without merging**